app/src/test/java/com/fcaputo/parkingapp/presenterTests/SettingsPresenterTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.fcaputo.parkingapp.settingsTests
+package com.fcaputo.parkingapp.presenterTests
```

Comment:
a package should never be in camel case. It should be just presenter as you are in the tests